build/charts/yorkie-cluster/templates/istio/ratelimit/config.yaml

```diff
@@ -0,0 +1,15 @@
+{{ if .Values.ratelimit.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ratelimit-config
```

Comment:
How about change this to `{{ .Values.ratelimit.name }}-config` to let user set resource name freely?